TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/AMainTeleOp.java

```diff
@@ -130,11 +130,11 @@ public void loop() {
         }
 
         /// Drives
-        boolean safeDrive = this.clawSlide.slideRotate.getLeftPosition() < 850 || this.clawSlide.claw.getLeftRotAngle() < 100;
-        if (safeDrive) {
-            this.drive.shift(this.gamepad1.left_stick_x, this.gamepad1.left_stick_y);
-            this.drive.rotate(this.gamepad1.right_stick_x * 0.6);
-        }
+//        boolean safeDrive = this.clawSlide.slideRotate.getLeftPosition() < 850 || this.clawSlide.claw.getLeftRotAngle() < 100;
+//        if (safeDrive) {
+        this.drive.shift(this.gamepad1.left_stick_x, this.gamepad1.left_stick_y);
+        this.drive.rotate(this.gamepad1.right_stick_x * 0.6);
+//        }
```

Comment:
```suggestion
        boolean safeDrive = true || this.clawSlide.slideRotate.getLeftPosition() < 850 || this.clawSlide.claw.getLeftRotAngle() < 100;
        if (safeDrive) {
            this.drive.shift(this.gamepad1.left_stick_x, this.gamepad1.left_stick_y);
            this.drive.rotate(this.gamepad1.right_stick_x * 0.6);
        }
```

Reply:
I was thinking that we will need to be able to drive while the claw is down, this is why I thought of turning safe drive off.

Reply:
yeah but make the condition always true is better than commend the if out, in styles

Reply:
What if I delete the commented code?

Reply:
hm okay